src/utils/PrivateRoute.js

```diff
@@ -1,15 +1,30 @@
 import React, { useContext } from "react";
-import { Redirect, Route } from "react-router-dom";
+import { Redirect, Route, Link } from "react-router-dom";
 
 import { IsAuthContext } from "./AuthContext";
 
 function PrivateRoute({ children, ...restOfProps }) {
-  const isAuth = true
-  
-  return isAuth ? 
-    (<Route {...restOfProps}>{children}</Route>)
-       :
-    (<Redirect exact to="/" /> )
+  const { isLoggedIn } = useContext(IsAuthContext);
+  const { isAuth } = useContext(IsAuthContext);
+
+  isLoggedIn();
```

Comment:
Wrap a `useEffect` around this function. It's best to not have these "floating" functions hanging around in your component. They should also be called by something. 
In this case a useEffect with an empty dependency array would be good

Reply:
It's complaining without dependency so now it has one